Rakefile

```diff
@@ -36,3 +36,35 @@ if File.exist?(build_defs_file)
     end
   end
 end
+
+desc "verify that commit messages match CONTRIBUTING.md requirements"
+task(:commits) do
+  %x{git log --no-merges --pretty=%s stable..$HEAD}.each_line do |commit_summary|
```

Comment:
Yeah, the easy/hacky answer is "have it different on the different branches". I don't know if there's something cleaner though.